packages/text/types.ts

```diff
@@ -20,11 +20,14 @@ export enum TextSize {
 }
 export type TextSizes = keyof typeof TextSize
 
+export type TextWeight = 400 | 500 | 700 | 800
```

Comment:
Why just those? Can we use all available? https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight

```
/* Keyword values */
font-weight: normal;
font-weight: bold;

/* Keyword values relative to the parent */
font-weight: lighter;
font-weight: bolder;

/* Numeric keyword values */
font-weight: 100;
font-weight: 200;
font-weight: 300;
font-weight: 400; /* normal */
font-weight: 500;
font-weight: 600;
font-weight: 700; /* bold */
font-weight: 800;
font-weight: 900;

/* Global values */
font-weight: inherit;
font-weight: initial;
font-weight: revert;
font-weight: revert-layer;
font-weight: unset;
```

Reply:
Btw it should be possible to get these values out of native element props

Reply:
Hm, looks like it's just `string | number`, maybe it makes sense to just use the same

<img width="493" alt="image" src="https://github.com/lidofinance/ui/assets/103929444/cbf1e824-f8af-4be5-9594-5fe2dacec347">

Reply:
`fontWeight` prop is not working for me
<img width="1091" alt="image" src="https://github.com/lidofinance/ui/assets/7289992/95bbcfaf-26bf-4411-a6e9-b60519ae2347">

Reply:
> Those I listed are only I have faced in the design system, another did not appeared before. As long as we are adopting design system as source for this ui-kit I see no need to add all other variability until we need them.

I don't think we should exclude valid props, because we haven't seen someone needed that before, this doesn't ensure no one will not need it or no one is using it on some other project right now 🤔 

> fontWeight prop is not working for me

There is no `fontWeight` property unless you add it. As for my example, that's just [codesandbox react app](https://codesandbox.io/p/sandbox/condescending-shannon-rz8n4d), I wanted to check what is the type for `<span style={{ fontWeight: value }}/>` so it's just `string | number`

Reply:
> I don't think we should exclude valid props, because we haven't seen someone needed that before, this doesn't ensure no one will not need it or no one is using it on some other project right now 🤔

I am not saying about that no one is using it, my thought was about taking ui/ux design system from figma as source of truth when we are defining set that we are will use in our kit. Imo we do not need to add options that designers are not defining in their layouts, because we will not use them too as long as we are referencing to their system. This is also limitation that will prevent appearance of custom styles that was not meant to be in our visual style by designers.


> There is no fontWeight property unless you add it. As for my example, that's just [codesandbox react app](https://codesandbox.io/p/sandbox/condescending-shannon-rz8n4d), I wanted to check what is the type for <span style={{ fontWeight: value }}/> so it's just string | number

Ok, I see, did not noticed you are talking about this exact type of prop. So your advice here to use `style` prop in cases when we need to define custom font-weight value?

Reply:
I'm totally agreed with that, the issue is that I'm not really sure if we are using the same Figma components everywhere, e.g. there are no outdated designs or there are no newer design components. We don't have such establish process, so it feels like it's safer to allow end users to be able to specify any value they can without the need to somehow overwrite ui-kit.

I guess, we can use type as `400 | 500 | 600 | ... | string | number`, it will still allow using any other value if needed, but at the same time it will guide ui-kit consumers, which values are expected. What do you think?

> Ok, I see, did not noticed you are talking about this exact type of prop. So your advice here is to use style prop in cases when we need to define custom font-weight value?

Nah, I was just curious what is "default" type for font-weight in style props for native elements

Reply:
> I guess, we can use type as 400 | 500 | 600 | ... | string | number, it will still allow using any other value if needed, but at the same time it will guide ui-kit consumers, which values are expected. What do you think?

Sure, I pushed this approach you suggested 👌